content/docs/contribute/atmosphere-pro/1-introduction/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: codelab
+title: Introduction
+description: Requirements for contributing to Atmosphere Pro
+draft: false
+order: 1
+---
+
+In this tutorial, we will be cloning our project so that you can make contributions to our end-to-end encrypted file transfer app, atmosphere pro. To get started, you will need the following:
+
+1. [Git](https://git-scm.com/downloads)
+2. A [GitHub](https://github.com/) account
+3. A code editor like [VSCode](https://code.visualstudio.com/download) for coding in Dart/Flutter.
+4. The [Flutter](https://docs.flutter.dev/get-started/install) framework
+5. An environment to test and run the application ([Android](https://developer.android.com/studio?gclid=CjwKCAjwx7GYBhB7EiwA0d8oe3wbfdyIU9rVXgxc6VwabUpnLdjnhEZHKxGupUw97lF91AZkbbNx9BoCB-cQAvD_BwE&gclsrc=aw.ds), [iOS](https://developer.apple.com/xcode/), [macOS](https://docs.flutter.dev/get-started/install/macos#macos-setup), or [Windows](https://docs.flutter.dev/get-started/install/windows)).
```

Comment:
Please remove the parameters from the Android Studio link

Reply:
Done in https://github.com/atsign-foundation/docs.atsign.com/pull/124/commits/d92f9a1ad8991954d0a5c2503e13d4baa01a1b38